sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -90,29 +95,34 @@ func init() {
 // New flags that are already put into pipeline options
 // should be added to this map.
 var flagFilter = map[string]bool{
-	"dataflow_endpoint":              true,
-	"staging_location":               true,
-	"worker_harness_container_image": true,
-	"labels":                         true,
-	"service_account_email":          true,
-	"num_workers":                    true,
-	"max_num_workers":                true,
-	"disk_size_gb":                   true,
-	"disk_type":                      true,
-	"autoscaling_algorithm":          true,
-	"zone":                           true,
-	"network":                        true,
-	"subnetwork":                     true,
-	"no_use_public_ips":              true,
-	"temp_location":                  true,
-	"worker_machine_type":            true,
-	"min_cpu_platform":               true,
-	"dataflow_worker_jar":            true,
-	"worker_region":                  true,
-	"worker_zone":                    true,
-	"teardown_policy":                true,
-	"cpu_profiling":                  true,
-	"session_recording":              true,
+	"dataflow_endpoint":                true,
+	"staging_location":                 true,
+	"worker_harness_container_image":   true,
+	"labels":                           true,
+	"service_account_email":            true,
+	"num_workers":                      true,
+	"max_num_workers":                  true,
+	"number_of_worker_harness_threads": true,
```

Comment:
This list is largely to filter out things that already show up visibly in the UI when set. If it's not UI visible by default, we probably shouldn't filter them out.

My recommendation here is *not* adding anything to start and we can filter them if we notice they stutter in the UI. Ultimately, this helps users (and us) know the configuration of a job, and failing in a "it shows up twice" is fine, but hiding the information could be painful.

I'll also note that if you tried these things already and noticed the stutter, then this comment can be disregarded.

Reply:
Yeah, that's a good call. I removed the new items